### DISCUSSION TITLE
A new open-source using shell-operator

### DISCUSSION THREAD

**Discussion by swnuom** [flant/shell-operator#561] [Show and tell]
Hi,
We are happy to inform you that we have recently released ClusterSlice (https://github.com/swnrg/clusterslice), a new open-source project using the shell-operator.

ClusterSlice is an open-source solution for large-scale, kubernetes-centered experimentation. It supports a declarative specification of experimentation slices, their deployment over multiple test-beds and domains, the utilization of heterogeneous physical and virtual resources, as well as multiple kubernetes flavors, network plugins, and plug-and-play application features.

It introduces well-designed abstractions that reduce experimentation complexity with improved reliability and reproducibility.

You can check the teaser video at: https://asciinema.org/a/uqWSBv3JBv0X3wdvfzgSoIe26

Thank you for your great work.

Regards,
Lefteris

**Comment by nabokihms** [comment]
@swnuom, that is awesome! Thanks for sharing.
